src/main/java/com/google/plantasy/utils/Game.java

```diff
@@ -8,7 +8,7 @@
     private Entity entity;
 
     public Game(Entity game_entity) {
-        if (game_entity.getKind().equals(GAME_KIND)) {
+        if (!(game_entity.getKind().equals(GAME_KIND))) {
```

Comment:
You shouldn't need the extra set of (), just the ! should sufice.

Reply:
Got it, those are gone.